File: URI/1 - INICIANTE/C/1094 - Experiencia.c
#include <stdio.h>

int main()
{
	int total = 0, z =0, x, qtd;
	float sapos = 0.0,ratos = 0.0, coelho = 0.0;
	char tipo;
	
	scanf(" %d", &x);
	
	while(z < x)
	{
		scanf(" %d %c", &qtd, &tipo);
		if(tipo == 'C')
		{
			coelho += qtd;
		}else if(tipo == 'R')
		{
			ratos += qtd;
		}else{
			sapos += qtd;
		}
		total += qtd;
		z++;
	}
	printf("Total: %d cobaias\n",total);
	printf("Total de coelhos: %.0f\n",coelho);
	printf("Total de ratos: %.0f\n",ratos);
	printf("Total de sapos: %.0f\n",sapos);
	printf("Percentual de coelhos: %.2f %%\n", (coelho*100.00)/total);
	printf("Percentual de ratos: %.2f %%\n", (ratos*100.00)/total);
	printf("Percentual de sapos: %.2f %%\n", (sapos*100.00)/total);
	
	return 0;
}

File: URI/1 - INICIANTE/Python/1094 - Experiencia.py
sapos = ratos = coelho = total = 0
z = 0
x = int(input())
while z < x:
    qtd, tipo = input().split(' ')
    qtd = int(qtd)
    if tipo == 'S':
        sapos += qtd
    elif tipo == 'R':
        ratos += qtd
    else:
        coelho += qtd
    total += qtd
    z += 1
print('Total: {} cobaias'.format(total))
print('Total de coelhos: {}'.format(coelho))
print(('Total de ratos: {}'.format(ratos)))
print('Total de sapos: {}'.format(sapos))
print('Percentual de coelhos: {:.2f} %'.format((coelho*100)/total))
print('Percentual de ratos: {:.2f} %'.format((ratos*100)/total))
print('Percentual de sapos: {:.2f} %'.format((sapos*100/total)))